run_gene_annotation_skiperror.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#SBATCH -J 500_ncbi_redgenes
```

Comment:
A concern here is the script will only work in a specific user account. And I don't think any of the changes here or in the PR affect the library code? We can chat more later this week if helpful

Reply:
I must have forgotten to git add gene_annotation.py. Thank you for catching that. I included them in the new push. And yes let's chat about how to generalize the bash script for more user account. Thanks!